source/process/spec-process.rst

```diff
@@ -0,0 +1,46 @@
+Spec Review Process
+===================
+
+This is a rough guideline for the process for drafting and reviewing specs. It’s meant to be used as a starting point, but not all specs will follow this exact process - please use your judgment, as it’s something that will vary depending on the situation.
```

Comment:
Maybe 

```
This is a rough guideline for drafting and reviewing specs. It’s meant to be used as a starting point, and not all specs will follow this exact process - please rely on your judgment, as not all steps may apply.
```

 - `for the process` seems unnecessary
 - `but` --> `and`
 - shortening the ending a little